src/main/java/net/comments/objects/NewCommentBody.java

```diff
@@ -0,0 +1,11 @@
+package net.comments.objects;
+
+public interface NewCommentBody {
```

Comment:
Please rename this to `ModifyCommentPage` as this page is the same for editing and creation.